# dropped code leeds to circular definition for >=sys-libs/musl-1.2.3 and
# is not needed as all our compilers support c++11
diff --git a/src/logbook/table.cxx b/src/logbook/table.cxx
index 6deee6d..8011d02 100644
--- a/src/logbook/table.cxx
+++ b/src/logbook/table.cxx
@@ -40,16 +40,6 @@
 #define DAMAGE_HEADER  FL_DAMAGE_ALL
 #define DAMAGE_ROWS  FL_DAMAGE_ALL
 
-/* 
- * nullptr is not available in gcc < 4.6
- * Redefine to NULL for CentOS 6 and OSX Darwin
- */
-#ifndef __FreeBSD__
- #ifndef nullptr
- #define nullptr NULL
- #endif
-#endif
-
 
 /*
  * ======================================
